src/NossoMercadoLivreAPI.Infra.Data/UnitOfWork/UnitOfWork.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using NossoMercadoLivreAPI.Domain.Interfaces.UnitOfWork;
+using NossoMercadoLivreAPI.Infra.Data.Context;
+
+namespace NossoMercadoLivreAPI.Infra.Data.UnitOfWork
+{
+    public class UnitOfWork : IUnitOfWork
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Como isso já é pronto pelo framework, eu sugiro remover.